### ISSUE TITLE
ERROR [user/pytorch-mpi internal] 

### ISSUE THREAD

**Issue by JayanthRR** [epfml/federated-learning-public-code#6]
Hello, Thanks for sharing your code. I am trying to install the docker image but I run into this issue. Can you please guide me on how to get the environment to work correctly? Thanks in advance. 

``` 
=> ERROR [user/pytorch-mpi internal] load metadata for docker.io/user/base:latest                                       0.3s
 => CANCELED [user/base internal] load metadata for docker.io/nvidia/cuda:11.2.1-cudnn8-devel-ubuntu18.04                0.2s
------
 > [user/pytorch-mpi internal] load metadata for docker.io/user/base:latest:
------
failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```

**Comment by Hangyul-Son** [comment]
Hello. I am facing the same issue. The problem seems to arise from not having the public key updated under the update from Nvidia. It would be greatly appreciated if the update it made. 
<img width="1440" alt="Screen Shot 2022-12-22 at 13 18 38" src="https://user-images.githubusercontent.com/47938735/209062434-42c95340-de1a-472c-bf38-3f2c3a16e6b4.png">

Thank you for the code.

**Comment by tlin-taolin** [comment]
fixed in 3502744fddefa0b7e9b87bb7b457e6b6438e20c6

**Comment by Hangyul-Son** [comment]
@tlin-taolin Thanks a lot!
